Documentation/Installation/Install.rst

```diff
@@ -71,6 +71,11 @@ At the root level of your web server, execute the following command:
             # Fetch a basic TYPO3 installation and its dependencies
             ddev composer create "typo3/cms-base-distribution:^13"
 
+..  tip::
+    The following commands expect a completely empty directory. Do not open the project in an
+    IDE like PhpStorm before the commands have been executed. IDEs will usually create a hidden folder like 
+    :path:`.idea` that will cause an error message with the `composer create-project` command.
```

Comment:
This is another command than the command from above. What is the difference between "create" and "create-project"

Reply:
Ack

Reply:
Strangely the ddev command says composer create and the non-ddev create-project... Do you have any idea which one would be correct?

Reply:
"create" is AFAIK just a shorthand alias for "create-project". Officially documented is only the latter. We should only use "create-project" then.

Reply:
Please use `create`.
https://ddev.com/blog/getting-started-with-ddev-and-composer/
See section "Built-in Composer commands in DDEV"
`create-project` will execute the original command from Composer and THAT needs a completely empty directory.
`create` is a specific command provided by DDEV and allows creating projects also in non empty environments

Reply:
So my sentence is correct then, `composer create-project` will fail when executed from an ide that already opened that directory? `create` can only be used with ddev

Reply:
Correct. I just stumbled across the "composer create-project" that suddenly appeared.

Maybe I miss something like:
There is another ddev command called `ddev composer create-project`, which will call the original `composer create-project` command which expects a complete empty directory. Else it will fail. So, please use the `ddev composer create` command as mentioned above, to allow installations in non-empty directories with f.e. `.idea` in case of PhpStorm.

Reply:
Not everyone is using DDEV and those without don't have the ddev composer create command